index.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+session_start();
+
+if (!$_SESSION['loggedIn']) {
+    header('Location:login.php');
+}
+
+require_once ('vendor/autoload.php');
+
+use Pokedex\DbConnection;
+use Pokedex\PokeList;
+
+$dbConnection = new DbConnection();
+$pokeList = new PokeList($dbConnection->getDB());
+
+$userID = (int)$_SESSION['id'];
+
+
+if (!empty($_POST)) {
```

Comment:
There's a lot of code in this if statement, should this be in an object?

Reply:
I did make an object initially, but I took it out since I was putting in data I already had and retrieving it immediately after. Broken up into a function in be4c2b4